builds/opendingux/gcw-zero/run.sh

```diff
@@ -1,11 +1,25 @@
 #!/bin/sh
-cd `dirname $0`
+SOURCE="$1"
+DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
```

Comment:
I don't understand your question. $1 can not be empty because you need to choose a file in the file browser. Plus, even if $1 is not passed, $HOME always redirects to /usr/local/home, it doesn't matter.